### ISSUE TITLE
Please document `mint` and `maxt` in `storage.Queryable`

### ISSUE THREAD

**Issue by pstibrany** [prometheus/prometheus#6784]
This is a request for a better documentation on `storage.Queryable` interface.

It has single `Querier(ctx context.Context, mint, maxt int64)` method, but it's not clear how `mint` and `maxt` passed to the `Querier` relate to `*SelectParams` parameter (and its `Start` and `End` fields) supplied to the `Select` or `SelectSorted` on the returned `Querier`.

Thank you.

(This was first asked in https://github.com/prometheus/prometheus/pull/6777/, but moved to a separate issue)

**Comment by bboreham** [comment]
Hello from the bug-scrub @pstibrany: it looks like the interface documentation didn't add this information yet.
Perhaps with 4 years' extra experience you gained some idea what the parameters mean, and can supply a PR?

**Comment by abhinandan-thakur** [comment]
Hi, Shouldn't this be closed since #19294  is merged.

**Comment by bboreham** [comment]
Yes, for that to happen automatically the line has to read just as in the template:

Usage: `Fixes #<issue number>`

I edited #19294 so you can see.

Thanks!
